RancherProjectManager/RancherProjectManagement.py

```diff
@@ -94,4 +81,25 @@ def process_namespace(self, namespace: V1Namespace):
         # Patch the project ID on there
         logging.info(f'Annotating namespace {namespace.metadata.name} for requested project named {project_name} with its ID {project_id}')
         annotations[self.project_id_annotation] = project_id
-        self.kubeapi.patch_namespace(namespace.metadata.name, namespace)
\ No newline at end of file
+        self.kubeapi.patch_namespace(namespace.metadata.name, namespace)
+    
+    def handle_project_role(self, namespace: str, project_id: str, rolename: str, members: List[str]):
+        resolved_members = []
+        for member in members:
+            resolved_member = self.rancher.search_principal(member)
+            if resolved_member is None:
+                logging.warning(f'Could not find a user or group in Rancher matching \"{member}\" for namespace {namespace}')
+                continue
+            resolved_members.append(resolved_member)
+        existing_members = self.rancher.get_project_members(project_id, rolename)
+        
+        new_memers = set(resolved_members).difference(existing_members)
```

Comment:
Did you go with `memers` in these intentionally?

Reply:
HA nope, good catch, gonna fix that before I merge